src/Stratis.Features.SQLiteWalletRepository/SQLiteWalletRepository.cs

```diff
@@ -1002,6 +1000,7 @@ private bool StartBatch(ProcessBlocksInfo round, ChainedHeader header)
                 round.PrevTip = (header.Previous == null) ? new HashHeightPair(0, -1) : new HashHeightPair(header.Previous);
                 round.NewTip = null;
                 round.Trackers = new Dictionary<TopUpTracker, TopUpTracker>();
+                round.BatchDeadline = DateTime.Now.Ticks + 10 * 10_000_000;
```

Comment:
What is `10_000_000`? Maybe it should be a constant for easier search and self-explanation